pegasus/sites.v3/code.org/public/js/volunteer-local.js

```diff
@@ -228,34 +244,24 @@ function loadMap(locations) {
 
   gmap.Load(mapOptions);
 
+  // Mapbox specific.
+
   if (window.location.search.indexOf("mapbox") !== -1) {
     $("#mapbox-container").show();
 
+    $("#mapbox-listings").html("");
+
     mapboxMap = new mapboxgl.Map({
       container: "mapbox",
       style: "mapbox://styles/codeorg/cjyudafoo004w1cnpaeq8a0lz",
       center: [lng, lat],
-      zoom: 12
+      zoom: 1
```

Comment:
Why change the zoom here?

Reply:
This was so that map would initially show the world world, rather than zoomed into the middle of the US.  It quickly pans/zooms to the desired view, though, so I don't have strong feelings about this.

Reply:
I was just curious, seems fine to me

Reply:
Thanks.  We will learn more once we play with it on production!